src/dao/AppDAO.js

```diff
@@ -151,7 +155,19 @@ class AppDAO extends AbstractContractDAO {
     };
 
     getLOCs = (account: string) => {
```

Comment:
Because the Chronomint contract will only be for LOCs management.
I can rename AppDAO to LOCsManagerDAO, if you want.